ui/app/styles/components/console-ui-panel.scss

```diff
@@ -146,8 +146,13 @@
 }
 
 .console-close-button {
-  position: absolute;
+  position: sticky;
   top: $spacing-xs;
-  right: $spacing-xs;
+  display: flex;
```

Comment:
There are some helper classes for flex things. "is-flex" and "is-flex-end". Again, up to you.

Reply:
Sounds good. I'm not sure I see the benefit since there is a class already for the element. I think if this element only needed to be flexed I would definitely use helpers instead.

Reply:
Agreed, unless you were having issues with overriding (most helpers use !important) if a class exists then helpers are not as relevant.